src/serializers/jobs-serializer.js

```diff
@@ -0,0 +1,80 @@
+import { Serializer as JsonApiSerializer } from 'jsonapi-serializer';
+import _ from 'lodash';
+
+import { contrib } from 'db/oracledb/contrib/contrib';
+import { formatSubObjects } from 'utils/format-sub-objects';
+import { serializerOptions } from 'utils/jsonapi';
+import { openapi } from 'utils/load-openapi';
+import { apiBaseUrl, resourcePathLink, paramsLink } from 'utils/uri-builder';
+
+const jobResourceProp = openapi.components.schemas.JobResult.properties.data.properties;
+const jobResourceType = jobResourceProp.type.enum[0];
+const jobResourceAttributes = jobResourceProp.attributes.allOf;
+const jobCombinedAttributes = _.merge(jobResourceAttributes[0], jobResourceAttributes[1]);
+const jobResourceKeys = _.keys(jobCombinedAttributes.properties);
+
+const prepareRawData = (rawJobs) => {
+  _.forEach(rawJobs, (job) => {
+    job.jobId = `${job.positionNumber}-${job.suffix}`;
+    job.accruesLeaveInd = job.accruesLeaveInd === 'Y';
+    job.classifiedInd = job.classifiedInd === 'Y';
+    job.employmentType = job['employeeClassification.code'] === 'XA' ? 'student' : 'graduate';
```

Comment:
So actually we only allow users to search if they are a student employee or not. `graduate` is just a placeholder for this case. I'm thinking we can either make this field as a boolean to `studentEmployeeInd` and also change the filter to boolean. Thought?

Reply:
Sounds good